cmd/evm/runner.go

```diff
@@ -236,19 +226,6 @@ func runCmd(ctx *cli.Context) error {
 		},
 	}
 
-	if cpuProfilePath := ctx.String(CPUProfileFlag.Name); cpuProfilePath != "" {
```

Comment:
How come you decided to remove this? I know it's not "generally useful", but from time to time it's pretty interesting to take a profile of a state-test execution, or e.g when running the snailtracer.

Reply:
The feature is still there, this is just handled by internal/debug now (through `debug.Setup` in the `App.Before` hook).

Reply:
Pretty good you made this comment though, because I noticed we were missing a call to the `debug.Exit` hook that actually writes the profile.

Reply:
It works like this btw:

```
./evm --pprof.cpuprofile cpu.out run --codefile code.bin
```